packages/mergeOnGreen/README.md

```diff
@@ -0,0 +1,21 @@
+mergeOnGreen
+
+Instructions are provided in [googleapis/repo-automation-bots](https://github.com/googleapis/repo-automation-bots/blob/master/README.md) for deploying and testing your bots.
+
+This bot uses nock for mocking requests to GitHub, and snap-shot-it for capturing responses; This allows updates to the API surface to be treated as a visual diff, rather than tediously asserting against each field.
+
+Running tests:
```

Comment:
I'm being very pedantic, but it would be nice to go back and make it so we generate a slightly nicer README out of the gate, an `h1` header is represented as `# My Header`, an `h2` as `## My Second Level Header`, I think we could make this look a bit better with these updates.